base/operators.jl

```diff
@@ -163,8 +163,9 @@ const ≠ = !=
     ≡(x,y) -> Bool
 
 Determine whether `x` and `y` are identical, in the sense that no program could distinguish
-them. Compares mutable objects by address in memory, and compares immutable objects (such as
-numbers) by contents at the bit level. This function is sometimes called `egal`.
+them. First it compares the types of `x` and `y`. If those are identical, it compares mutable
```

Comment:
"First it compares .... If those are identical, it compares ..." -> "First compare .... If those are identical, compare ..." ?

Reply:
Perhaps
> First the types of `x` and `y` are compared. If those are identical, mutable objects are compared by address in memory and immutable objects (such as numbers) are compared by contents at the bit level.

I actually think that sounds slightly better than the imperative in this case but I don't care that much.

Reply:
Both are clear for me (I am not a native speaker though).